cmake/projects/lehrfempp/hunter.cmake

```diff
@@ -13,11 +13,11 @@ hunter_add_version(
     PACKAGE_NAME
     lehrfempp
     VERSION
-    0.7.3
+    0.7.5
```

Comment:
Please add new version, not update existent.

Reply:
I thought I would replace it because 0.7.3 is probably never going to be used. But I've added it now, just to stay consistent...